dls-controls/malcolmjs
# Feature/widget table misc

## Description

Added ability to add row to writeable tables, in addition to properly implementing dirty flag and associated behaviour for tables. Also added changes from outstanding pull request #190 

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- npm start
- navigate to gui/PANDA:SEQ1
- click table view button
- try editing the table!

## Agile board tracking

connect to #155 
connect to #161 
connect to #162
